src/http/httpd.cc

```diff
@@ -183,11 +183,24 @@ set_request_content(std::unique_ptr<http::request> req, input_stream<char>* cont
     }
 }
 
+static void set_keep_alive_header(http::reply& resp, bool keep_alive) {
+    if (keep_alive) {
+        if (resp._version == "1.0") {
+            resp.add_header("Connection", "Keep-Alive");
```

Comment:
Looks good, but worthy of a comment: This case is NOT standard HTTP 1.0, but rather a non-standard extension over HTTP 1.0 which is documented here https://datatracker.ietf.org/doc/html/rfc2068#section-19.7.1. 

Anyway, since this is correct and also appears to be pre-existing code that you moved, I can't complain.